more_itertools/tests/test_more.py

```diff
@@ -1105,11 +1105,11 @@ def test_basic(self):
         @contextmanager
         def managed():
             before.append(1)
-            yield 1
```

Comment:
I meant unique across all the values—1,2,3 vs. range(3)'s 0, 1, 2—but don't waste your time messing with it more.